test/androidTest/java/org/thoughtcrime/securesms/espresso/ConversationHelper.java

```diff
@@ -68,9 +67,8 @@ public ConversationHelper selectMessage(int position) {
         this.messageSelected = true;
 
         onView(withId(android.R.id.list))
-            .perform(closeSoftKeyboard())
             .perform(scrollToPosition(position))
-            .perform(actionOnItemAtPosition(position, clickPercent(0f,0f)));
+            .perform(actionOnItemAtPosition(position, longClickPercent(0f,0.9f)));
```

Comment:
this might not work on my cellphone, we might need to figure a dynamic solution in the future

Reply:
can you elaborate why you think it wouldn't?

Reply:
or you can just try

Reply:
I will